### PR TITLE
Bug 1176913 - focus current app on secure-appopened event

### PR DESCRIPTION
**Pull request by davidflanagan**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=555242c1335e57bbb9e6e32296fc00a5b5f17780

**Comment by davidflanagan** [comment]
closing and re-opening to retrigger the tests, since it didn't seem to happen automatically when I added tests to my PR.

**Comment by davidflanagan** [comment]
manually restarted GU27 test and verified that the new test added to secure_window_manager_test.js ran successfully

File: apps/system/test/unit/secure_window_manager_test.js [hunk 1]
```diff
@@ -88,6 +88,21 @@ suite('system/SecureWindowManager', function() {
...
 
+    test('App focused after opening', function() {
+      var stubFocus = this.sinon.stub(appFake, 'focus');
+      try {
+        window.secureWindowManager.handleEvent({type: 'secure-appcreated',
+                                                detail: appFake});
+        window.secureWindowManager.handleEvent({type: 'secure-appopened',
+                                                detail: appFake});
+        assert.isTrue(stubFocus.calledOnce,
+                      'the app was not given keyboard focus after opening');
```
Comment (snowmantw) [inline apps/system/test/unit/secure_window_manager_test.js]: nits: The indentation looks a bit weird. Could you just make it align with the two spaces? I know this is very personal but we should keep the file with a unified style.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e695172006effba291b516e6d4b7c79268c103bd

**Pull request merged**